### PR TITLE
Shopping cart

### LINKED ISSUE #19: Implement cart page layout
Create HTML template for the cart page.
*This does not include database functionality*

### LINKED ISSUE #20: Implement cart functionality
Using the template in:
- [x] #19 

Implement cart alteration functions (increment, decrement, remove etc.)

### LINKED ISSUE #21: Shopping cart
As a customer, I want to be able to browse products and add them to my shopping cart, because sometimes I want to buy more than one item at a time, and would like to be able to do so in a single order. As a logged in web user, items can be selected to go to the cart for future purchases.
- If you select “Remove”, the item will be removed from the cart.
- If you select “Add to cart”, the item will be added to the cart.
In all cases a confirmation message will appear explaining the modification that was done.

ACCEPTANCE TEST
- User can add and remove items to their cart.
- User receives a confirmation message of modifications done to his cart.
- User can view their cart items.

PRIORITY: Medium
STORY POINTS: 6
- [x] #18 
- [x] #19 
- [x] #20

### PR DESCRIPTION
**Pull request by Ariiiane**
image

Front end is finish, and functionality (back end) is to be implemented in the next task
Closes #19 
Closes #20 
Closes #21

### CODE REVIEW

**Review by AryanSh1380 — COMMENTED** [review]
After pulling ShoppingCart branch, the footer is now in the middle of the page and doesn't appear at the bottom. Moreover, when pressing on add to cart, nothing happens. Cart tab is also no redirecting anywhere

**Review by JR-prog — CHANGES_REQUESTED** [review]
Please add a link to /cart (from the header).
Also, the footer is also locked for me, and I cannot see most of the information in most pages. It also seems like some of the css is inconsistent (proceed to checkout button hard to read).

**Comment by Ariiiane** [comment]
Task: #19: Implement cart page layout

The cart link has been added, can you please provide a screen shot for the footer issue, I can't see them on any page.

**Comment by Ariiiane** [comment]
https://user-images.githubusercontent.com/53889883/159163958-a7e77426-c923-492b-b2c3-1a49ed6ba98a.mp4

Here is a video of what I see

**Review by JR-prog — CHANGES_REQUESTED** [review]
image

Footer issue gone now, but the styling for the proceed button still seems off? I think its because I am checking it on my laptop (smaller screen), so you might have to make it more responsive.

File: public/css/app.css [hunk 1]
```diff
@@ -56,6 +56,9 @@ body, html {
     background: #05497B;
     color:white;
     font-family: Arial, Helvetica, sans-serif;
+    position: absolute;
```
Comment (JR-prog) [inline public/css/app.css:59]: If this is absolute, then it doesn't work with smaller screen sizes (the footer will be in the middle of the content). 
image
(Sorry I didn't catch this problem on last review)
Either leave it as relative (which is uglier on bigger screens) or find another way to make it at the bottom only.

**Review by JR-prog — CHANGES_REQUESTED** [review]
Button looks good. Added an extra comment about the footer that I missed in the last review.

**Review by JR-prog — CHANGES_REQUESTED** [review]
The footer is still inconsistent in other pages. Worst case, can you just change it in the css for the cart page only?

**Comment by Ariiiane** [comment]
Task: #20 - implementation of cart functionality
This task is now implemented
image

**Comment by Ariiiane** [comment]
Add to cart and delete functionalit

https://user-images.githubusercontent.com/53889883/159380201-b945f0fa-f1a4-4c6b-ba6d-9c125e9b90bc.mp4

y

**Review by JR-prog — CHANGES_REQUESTED** [review]
Adding and removing from cart works for me!

However, the last item in the cart is always hidden by the footer.

**Comment by Ariiiane** [comment]
I fixed the footer by imposing a minimum height to the page

**Review by JR-prog — APPROVED** [review]
Works for me!

**Pull request merged**